bpf/bpf.go

```diff
@@ -360,6 +437,54 @@ func (b *BPFKProbePerf) readVersion() (int, error) {
 	return 0, nil
 }
 
+func utsnameStr(in []int8) string {
+	out := make([]byte, len(in))
+
+	for i := 0; i < len(in); i++ {
+		if in[i] == 0 {
+			break
+		}
+		out = append(out, byte(in[i]))
+	}
+
+	return string(out)
+}
+
+func currentVersion() (int, error) {
+	var buf syscall.Utsname
+	if err := syscall.Uname(&buf); err != nil {
+		return -1, err
+	}
+
+	releaseStr := strings.Trim(utsnameStr(buf.Release[:]), "\x00")
+
+	kernelVersionStr := strings.Split(releaseStr, "-")[0]
```

Comment:
@iaguis This does not work in the rkt kernel (rkt with the kvm flavor) because releaseStr is "4.8.6rkt-v1":
```
# uname -r
4.8.6rkt-v1
```

Symptoms:
```
strconv.ParseInt: parsing "6rkt": invalid syntax
```